### PR TITLE
Add seed docs for Azure

### PR DESCRIPTION
**Pull request by fabiim**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1743.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Restore a database backup](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1743.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Restore a database backup](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1743.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1743.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Review by renetapopova — COMMENTED** [review]
Thanks, @fabiim. I have rewritten the authentication page to style it like the other clouds. Could you please take a look to see if it is correct?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1743.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/partials/azb-credentials.adoc [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+For more information, see the Azure official documentation on link:https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal[Create a storage account].
+.. Create a blob container in the Azure portal. +
+For more information, see the Azure official documentation on link:https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal[Quickstart: Upload, download, and list blobs with the Azure portal].
+. Install the Azure CLI by following the instructions in the Azure official documentation -- link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli[Azure official documentation].
+. Authenticate the neo4j or neo4j-admin process against Azure using the default Azure credentials. +
+See the Azure official documentation on link:https://learn.microsoft.com/en-us/java/api/com.azure.identity.defaultazurecredential?view=azure-java-stable[default Azure credentials] for more information. +
+Alternatively, you use another supported authentication method.
+See the Azure official documentation on link:https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli[Authenticate with Azure CLI] for more information.
```
Comment (fabiim) [inline modules/ROOT/partials/azb-credentials.adoc:10]: This isn't very clear. The only supported methods are the ones referenced in the previous page ("default Azure credentials"). So this isn't an alternative, it is just one of the supported methods, the one I considered the simplest to use. 

We could also remove this phrase. It isn't essential, readers will be able to find this information based on the previous link provided

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1743.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1743.surge.sh

Updated pages:
- [Aggregate a database backup chain](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/aggregate)
- [Back up an offline database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/offline-backup)
- [Back up an online database](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Restore a database backup](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/backup-restore/restore-dump)
- [Consistency checker](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/consistency-checker)
- [Import](https://neo4j-docs-operations-1743.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by renetapopova** [comment]
Hey @fabiim, I removed the sentences. Do you think we can merge it now?

**Comment by fabiim** [comment]
@renetapopova yes, but you need to approve it first

**Review by renetapopova — APPROVED** [review]

**Pull request merged**